src/Leela.cpp

```diff
@@ -82,6 +82,8 @@ static void parse_commandline(int argc, char *argv[]) {
         ("logfile,l", po::value<std::string>(), "File to log input/output to.")
         ("quiet,q", "Disable all diagnostic output.")
         ("noponder", "Disable thinking on opponent's time.")
+        ("benchmark", "Test network and exit. Stop other processes and use "
+                      "-t1 and -s<#> for more reproducible results.")
```

Comment:
Is there any reason why we wouldn't simply always default seed=1 for this?

Reply:
Done.

I didn't think that seed=1 would matter if we didn't also default to 1 thread. Updated thread default so that we can tell when user did not pass in a thread (before with the default value it was impossible to tell whether the user passed in the default value or it was just set by default).